### DISCUSSION TITLE
소셜 로그인시 social_service_id 검증

### DISCUSSION THREAD

**Discussion by Dae-Hwa** [Sc0up/scoup-backend#182] [Troubleshooting docs]
`social_service_id` 검증이 필요할까요?
현재는 아무 값이나 들어가도 회원가입이 되는 것 같네요.
임의의 사용자가 저희 클라이언트를 거치지 않고 직접 post 요청을 보내면 우회해서 회원가입이 가능하다는건데, 악용할 수 있는 방법은 뭐가 있을까요

**Comment by Dae-Hwa** [comment]
![image](https://user-images.githubusercontent.com/24666330/137072543-8ef33d86-7879-4013-a8bb-90910f7ab47b.png)

만약 임의의 사용자가 운좋게 특정 id와 일치하는 값을 넣으면 해당 소셜계정 사용 불가능한 경우가 생길 수 있다.

세션에 넣어 버리는 방안 고려 중

**Reply by janeljs** [reply]
ThreadLocal을 사용해도 좋을 것 같아요ㅎㅎ
https://javacan.tistory.com/entry/ThreadLocalUsage
